src/OrchardCore.Modules/OrchardCore.ContentPreview/ResourceManifest.cs

```diff
@@ -0,0 +1,17 @@
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.ContentPreview
+{
+    public class ResourceManifest : IResourceManifestProvider
+    {
+        public void BuildManifests(IResourceManifestBuilder builder)
+        {
+            var manifest = builder.Add();
+
+            manifest
+                .DefineScript("oc-contentpreview-edit")
+                .SetUrl("~/OrchardCore.ContentPreview/Scripts/contentpreview.edit.min.js", "~/OrchardCore.ContentPreview/Scripts/contentpreview.edit.js")
+                .SetVersion("1.0");
```

Comment:
Can you put this to version `1.0.0` I think we have trouble sometimes if it doesn't have a minor version.

And call it `contentpreview-edit` rather than `oc-` - I don't think we're prefixing any other scripts with oc?

Reply:
updated